### ISSUE TITLE
this app doesn't work

### ISSUE THREAD

**Issue by nantongxue** [alievk/avatarify-desktop#2]
I don't know the reason
https://youtu.be/Oi6OByxe5DM

**Comment by JohanAR** [comment]
That was an original way to do a bug report :) Unfortunately I don't know how to debug avatarify-desktop yet (I support the python version because I use Linux) but I will summarize your issue for anyone else looking at it:

Avatarify app is installed and it starts, but the preview is empty/white. The camera appears as "DS-2UCTV38W-S" in Avatarify, and it is demonstrated to work in a different capture program

**Comment by staddy** [comment]
The issue is most likely about unsupported camera codec.
The conversion code is here.
https://github.com/alievk/avatarify-desktop/blob/559924fe6b62312c3155c1216c68ccc9afec1b00/avatarify/src/camera/AsyncCameraCapture.cpp#L73-L92

**Comment by nantongxue** [comment]
> That was an original way to do a bug report :) Unfortunately I don't know how to debug avatarify-desktop yet (I support the python version because I use Linux) but I will summarize your issue for anyone else looking at it:
> 
> Avatarify app is installed and it starts, but the preview is empty/white. The camera appears as "DS-2UCTV38W-S" in Avatarify, and it is demonstrated to work in a different capture program

Yes, "DS-2UCTV38W-S" is my webcam, it works in other programs properly

**Comment by nantongxue** [comment]
> The issue is most likely about unsupported camera codec.
> The conversion code is here.
> https://github.com/alievk/avatarify-desktop/blob/559924fe6b62312c3155c1216c68ccc9afec1b00/avatarify/src/camera/AsyncCameraCapture.cpp#L73-L92

i'm not a professional programmer, so this means i should change some code in somewhere? but this is a "exe" program

**Comment by JohanAR** [comment]
You can download the complete source code from github, modify it and compile your own exe. But you need to learn at least the basics of programming first
